pages/docs/overview/faqs.mdx

```diff
@@ -0,0 +1,61 @@
+# General FAQ
+
+#### What are the benefits of using Tangle Network for asset transfers?
+
+Tangle Network employs connected shielded pool protocols and zero-knowledge proofs to ensure privacy and security during asset transfers. It enables users to privately move and transfer assets between blockchains while obfuscating transaction history.
+
```

Comment:
I think we should make it clear that Tangle network is facilitating the MPC protocol and governance system, specifically.